SConstruct

```diff
@@ -170,23 +170,68 @@ opts = Variables(customs, ARGUMENTS)
 # Target build options
 opts.Add("platform", "Target platform (%s)" % ("|".join(platform_list),), "")
 opts.Add("p", "Platform (alias for 'platform')", "")
-opts.Add(EnumVariable("target", "Compilation target", "editor", ("editor", "template_release", "template_debug")))
-opts.Add(EnumVariable("arch", "CPU architecture", "auto", ["auto"] + architectures, architecture_aliases))
+opts.Add(
+    EnumVariable(
+        "target",
+        "Compilation target",
+        "editor",
+        ("editor", "template_release", "template_debug"),
+    )
+)
```

Comment:
Sounds like you ran `black` without `-l 120`, so it used a lower limit for the line length. Those changes should be reverted.

Reply:
Done! Thanks!